src/lycophron/cli.py

```diff
@@ -92,6 +93,218 @@ def configure():
     pass
 
 
+@lycophron.command()
+@click.option("--filename", prompt="Filename", default="data.xlsx")
+@click.option("--journal", default=False, is_flag=True)
+@click.option("--meeting", default=False, is_flag=True)
+@click.option("--imprint", default=False, is_flag=True)
+@click.option("--thesis", default=False, is_flag=True)
+@click.option("--dwc", default=False, is_flag=True)
+@click.option("--gbif-dwc", default=False, is_flag=True)
+@click.option("--ac", default=False, is_flag=True)
+@click.option("--dc", default=False, is_flag=True)
+@click.option("--openbiodiv", default=False, is_flag=True)
+@click.option("--obo", default=False, is_flag=True)
```

Comment:
comment: we could group these into something like `--custom=dwc,ac,obo`, since there can be an arbitrary number of custom field namespaces added in the future.